packages/ui-components/src/components/range/range.tsx

```diff
@@ -49,10 +49,9 @@ export class KvRange implements IRange, IRangeEvents {
 			<Host>
 				<div class="range-container">
 					<input id="slider" class="slider" type="range" min={this.min} max={this.max} value={this.value} step={this.step} onInput={this.applyCssStyles} />
-					<div id="selector" class="selector">
-						<div class="select-btn"></div>
-						<span class="select-value">{this.value}</span>
-					</div>
+					<span id="select-value" class="select-value">
```

Comment:
This div was here due to the selector button design defined in Zeplin. I believe that with this approach you removed the style from the selector button and now the button does work because it is the default range selector from each browser. Please confirm and dont forget to follow the design established in zeplin 😃